src/main/java/com/accelad/acctive/sim/kernel/core/evaluator/diff/BetweenStrictlyLessThanMaxFunction.java

```diff
@@ -30,7 +30,7 @@ public X getValue() {
         double a = firstBound.getValue().getReal();
         double b = secondBound.getValue().getReal();
 
-        if (v >= Math.min(a, b) && v <= Math.max(a, b)) {
+        if (v >= Math.min(a, b) && v < Math.max(a, b)) {
             return DFFactory.get(1d);
```

Comment:
It is exactly the same !
We can modify it but it does not seem to be in the scope !